PacketTunnelProvider/NetworkProtection/NetworkProtectionPacketTunnelProvider.swift

```diff
@@ -285,11 +285,16 @@ final class NetworkProtectionPacketTunnelProvider: PacketTunnelProvider {
             if featureVisibility.shouldMonitorEntitlement() {
                 return { accountManager.accessToken }
             }
-            return { nil } }()
+            return { nil }
+        }()
+#if os(macOS)
         let tokenStore = NetworkProtectionKeychainTokenStore(keychainType: .dataProtection(.unspecified),
                                                              errorEvents: nil,
                                                              isSubscriptionEnabled: true,
                                                              accessTokenProvider: accessTokenProvider)
+#else
```

Comment:
We can probably simplify this down to only use the `else` case since the `if` case will never be true, right?